scalalib/src/mill/scalalib/JavaModule.scala

```diff
@@ -36,7 +36,8 @@ trait JavaModule
     with SemanticDbJavaModule { outer =>
 
   def zincWorker: ModuleRef[ZincWorkerModule] = super.zincWorker
-
+  type JavaTests = JavaModuleTests
+  @deprecated("0.11.10, use JavaTests instead")
```

Comment:
```suggestion
  @deprecated("Use JavaTests instead", since = "mill 0.11.10")
```

Reply:
We use uppercase name everywhere.

```suggestion
  @deprecated("Use JavaTests instead", since = "Mill 0.11.10")
```